docs/guides/internal-mcp-credentialref.md

```
@@ -0,0 +1,124 @@
+
+## 3. Verify the Registration
+
+After creating the resource, verify that the MCP Gateway has successfully processed the registration and discovered the tools.
+
+```bash
+# Check the status of the registration
+kubectl get mcpserverregistration internal-api-key-server -n mcp-test
+```
+
+Example output:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)